backend/courses/tests.py

```diff
@@ -459,7 +459,8 @@ def test_mixed_other_already_populated(self):
 class ParseOpendataResponseTestCase(TestCase):
     def test_parse_response(self):
         upsert_course_from_opendata(
-            json.load(open("courses/test-opendata.json", "r"))["result_data"][0], TEST_SEMESTER
+            json.load(open("backend/courses/test-opendata.json", "r"))["result_data"][0],
```

Comment:
You should be able to make this relative by only specifying the file name and not the path since they're in the same directory. Platform does something similar in our org tests somewhere.

Reply:
The right way to do it is to use the OS path things and specify an absolute path... this is being weird since I'm still running tests from the project root with vscode but it shouldn't be tied to my workflow